taxi/models.py

```diff
@@ -3,6 +3,7 @@
 
 
 class Manufacturer(models.Model):
+    objects = object
```

Comment:
The `objects` attribute is incorrectly assigned as `object`. In Django models, `objects` is typically an instance of `models.Manager`. You should remove this line to avoid overriding the default manager.